src/helm/benchmark/presentation/summarize.py

```diff
@@ -158,6 +151,38 @@ def get_method_display_name(model_display_name: Optional[str], info: Dict[str, A
     return (model_display_name or "???") + (f" [{dict_to_str(info)}]" if len(info) > 0 else "")
 
 
+def compute_row_win_rates(table: Table) -> List[Optional[float]]:
+    """
+    Computes the average win rate of each row across columns. That is, for a given row, if we pick a random column and
```

Comment:
I think its clearer to say "we pick a column uniformly at random" since that makes clear random has a formal definition (i.e. it isn't the lay usage of random to mean "arbitrary").